### PR TITLE
resolved data fetch + render bugs

### PR DESCRIPTION
**Pull request by NoerGitKat**
You should be able to continue now 😉

### CODE REVIEW

File: src/custom-hooks/useGetData.js [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
+  )
+
+  const getData = useCallback(async () => {
+    // ==== firebase firestore realtime data update =====
+    onSnapshot(collectionRef, (snapshot) => {
+      const { docs } = snapshot
+      const mappedDocs = docs.map((doc) => ({ ...doc.data(), id: doc.id }))
+      return setData(mappedDocs)
+    })
+  }, [collectionRef])
```
Comment (Qoosim) [inline src/custom-hooks/useGetData.js:20]: 1. Wonderful work my mentor. I appreciate your time. I noticed getData is made to be asycn function, but I could not see `await` anywhere within the function. I want to know why that is so. Thanks.

**Review by Qoosim — COMMENTED** [review]
@NoerGitKat thanks a lot for helping me to fix this blockage. I appreciate you. :tada: :heavy_check_mark:

File: src/custom-hooks/useGetData.js (hunk 1 above)
Comment (NoerGitKat) [reply src/custom-hooks/useGetData.js:20]: I think I forgot to remove it, so please do so!

**Comment by Qoosim** [comment]
Alright, thanks

**Pull request merged**